app/controllers/admin_reports_controller.rb

```diff
@@ -0,0 +1,39 @@
+class AdminReportsController < ApplicationController
+  helper_method :attendance
+  helper_method :average
+  helper_method :absence
```

Comment:
change for this `helper_method :attendance, :average, :absence`

Reply:
ty, done